extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -858,12 +858,12 @@ export class AmpA4A extends AMP.BaseElement {
               extensionId => extensions.preloadExtension(extensionId));
           // Preload any fonts.
           (creativeMetaDataDef.customStylesheets || []).forEach(font =>
-            this.preconnect.preload(font.href));
+            this.preconnect.preload(font.href, 'font'));
```

Comment:
Should this also set `crossorigin` or is this done further below?

Reply:
I did some manual testing and found out these are actually stylesheets (that contain an `@font-face`). Fixed the type here.

I also did some manual testing where I modified an existing component (I didn't find any examples preloading the fonts themselves).

I added a call to preload a font, then appended a stylesheet defining that font, then referenced it via CSS. When I preload the font, everything works as expected and there is no extra round trip to get the font for both Safari and Chrome. When I remove the preload of the font, there is an extra round trip once I reference it via CSS.

Reply:
Hmm, I'm quite certain the `crossorigin` attribute is needed. Could we add it anyway?

Reply:
Basically

`if (as === 'font') preload.setAttribute('crossorigin', '')`